crates/eframe/src/epi/mod.rs

```diff
@@ -774,6 +774,11 @@ impl Frame {
         self.info.clone()
     }
 
+    /// Get `IntegrationInfo` as a reference.
+    pub fn info_ref(&self) -> &IntegrationInfo {
+        &self.info
+    }
```

Comment:
Let's just change the old function to return a reference instead. Users can just call `.clone()` themselves.